addons/circulation/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -71,108 +58,194 @@ class ACE_Medical_Treatment_Actions {
 
     #include "Blood_Medical.hpp"
 
-    class AED: CPR {
-        displayName = CSTRING(Defib_Action_Use);
-        displayNameProgress = CSTRING(AED_PROGRESS);
-        icon = QPATHTOF(ui\defib.paa);
+    class AnalyzeRhythm: CheckPulse {
+        displayName = CSTRING(AnalyzeRhythm);
+        displayNameProgress = "";
+        allowedSelections[] = {"Body"};
+        icon = QPATHTOF(ui\icon_aed_shock.paa);
+        treatmentTime = 0.01;
+        items[] = {};
+        consumeItem = 0;
+        condition = QUOTE([ARR_2(_medic, _patient)] call FUNC(Defibrillator_CheckCondition));
```

Comment:
```suggestion
        condition = QFUNC(Defibrillator_CheckCondition);
```
it can be just QFUNC because parameters are passed in same order as in that array